### PR TITLE
PLANET-3033: Carousel: use h1 for title for SEO reasons

### PR DESCRIPTION
**Pull request by comzeradd**
This is just a simple heading substitution. Visually it should have no effect.

### CODE REVIEW

**Comment by pablocubico** [comment]
@comzeradd hey Nikos, this breaks the indicator positioning in the new carousel:

image

Here is the offending line:

https://github.com/greenpeace/planet4-plugin-blocks/blob/develop/assets/js/partials/carousel_header.js#L387

**Comment by comzeradd** [comment]
@pablocubico good catch! PR amended.

**Comment by pablocubico** [comment]
@comzeradd oops, I introduced a few conflicts here, can you check it? 😬

**Comment by comzeradd** [comment]
@pablocubico it should be ok now

**Comment by pablocubico** [comment]
@comzeradd  The indicators are misaligned: 

image

Probably because:
https://github.com/greenpeace/planet4-plugin-blocks/pull/480/files#diff-7e0cd6a667423f985e940a4874c3142eL391

File: assets/js/partials/carousel_header.js [hunk 1]
```diff
@@ -388,7 +388,7 @@ $(document).ready(function() {
     },
 
     positionIndicators: function() {
-      var $indicators = $('.carousel-indicators.carousel-indicators-large');
+      var $indicators = $('.carousel-indicators');
```
Comment (pablocubico) [inline assets/js/partials/carousel_header.js]: This should have the extra `.carousel-indicators-large` selector! I guess it was lost during conflict resolution.

Comment (comzeradd) [reply assets/js/partials/carousel_header.js]: Problem fixed ;)

**Pull request merged**